src/psyclone/tests/psyir/nodes/structure_member_test.py

```diff
@@ -124,3 +124,22 @@ def test_sm_child_validate():
     with pytest.raises(GenerationError) as err:
         smr.addchild(None)
     assert "'NoneType' can't be child 1 of" in str(err.value)
+
+
+def test_sm_member_property():
+    ''' Check the member property of StructureMember. '''
+    kschedule = nodes.KernelSchedule("kname")
+    grid_var = create_structure_symbol(kschedule.symbol_table)
+    assignment = nodes.Assignment(parent=kschedule)
+    grid_ref = nodes.StructureReference.create(grid_var, ['area', 'nx'],
+                                               parent=assignment)
+    smem_ref = grid_ref.member
+    assert isinstance(smem_ref, nodes.StructureMember)
+    assert isinstance(smem_ref.member, nodes.Member)
+    assert smem_ref.member.name == "nx"
+    # Break the node's children to check the exception
+    smem_ref._children = ["wrong"]
+    with pytest.raises(InternalError) as err:
+        smem_ref.member
```

Comment:
You can prepend `_ =` to avoid the pylint complain.

Reply:
Done.